### ISSUE TITLE
Maybe reference a benchmmark site ?

### ISSUE THREAD

**Issue by stonebig** [RobertoPrevato/BlackSheep#23]
I found this site :
https://www.techempower.com/benchmarks/

with a clother look at Python only framework there:
https://www.techempower.com/benchmarks/#section=data-r18&hw=ph&test=fortune&l=zijzen-7

... where I found blacksheep

**Comment by RobertoPrevato** [comment]
Hi @stonebig,
Thanks for your interest in my framework. :blush:
The benchmarks are already referenced: here in the main README with these words: _High performance, see results from TechEmpower benchmarks (links in Wiki page)_; and in more details in the [Wiki, here](https://github.com/RobertoPrevato/BlackSheep/wiki/Server-performance).

I am the one who made the PR to TechEmpower benchmarks to include my web framework.

**Note:** these benchmarks refer to a past version of BlackSheep, that included an implementation of HTTP Server. I later decided to separate the web framework from the HTTP Server component, I made BlackSheep ASGI compatible; so now it requires an ASGI HTTP Server to run.

This is a good decision because I can focus on high level features, and it's possible to deliver a good web framework working alone (as I do today); while reusing low level components that I wouldn't be able to deliver. For example support for HTTP/2; HTTP/1.1; HTTP/1.0 that can be found among [uvicorn](http://www.uvicorn.org/) and [hypercorn](https://pgjones.gitlab.io/hypercorn/).

In other words, benchmarks are slightly out of date; however the web framework is better than it used to be, because I corrected several issues; and the performance is still similar to what you see in TechEmpower, if you use BlackSheep together with Uvicorn.

Final note: I am now working on project templates for BlackSheep, and migrating one of my most complex projects to use my framework + Uvicorn, and I can say my framework is worth trying, it works nicely.

**Comment by stonebig** [comment]
ok, so closing.

**Comment by RobertoPrevato** [comment]
:+1:
